ansible/playbook.yml

```diff
@@ -270,6 +271,40 @@
       - pixelbook-touchscreen-click
       - pixelbook-keyboard-backlight
 
+    - name: Ensure keyboard hwdb is not overridden
+      become: true
+      file:
+        path: /etc/udev/hwdb.d/61-eve-keyboard.hwdb
+        state: absent
+      when: leftmeta == "capslock"
+
+    - name: Create keyboard hwdb override
+      become: true
+      copy:
+        src: /usr/lib/udev/hwdb.d/61-eve-keyboard.hwdb
+        dest: /etc/udev/hwdb.d/61-eve-keyboard.hwdb
+      when: leftmeta != "capslock"
+
+    - name: Set leftmeta to leftmeta in hwdb override
+      become: true
+      lineinfile:
+        path: /etc/udev/hwdb.d/61-eve-keyboard.hwdb
+        regexp: '^ KEYBOARD_KEY_db=capslock$'
+        line: ' KEYBOARD_KEY_db=leftmeta'
```

Comment:
Does the `leftmeta` in L293 evaluate to the string `"leftmeta"` or the _variable_ `leftmeta` if the former, then regardless of what the user specifies as override, if it's not `capslock` it'll set the override `leftmeta`.